lib/ex_okex/spot/private/list_accounts.ex

```diff
@@ -0,0 +1,11 @@
+defmodule ExOkex.Spot.Private.ListAccounts do
+  import ExOkex.Api.Private
+
+  @prefix "/api/spot/v3"
+  @path "#{@prefix}/accounts"
```

Comment:
Maybe move this inside `list_accounts/1` for consistency?

Reply:
The intention here was to avoid building the path on each call when they don't have a run time parameter. 

To make it consistent. It looks like I should also do it in `create_bulk_orders` & `create_order`